js/src/ui/publishBox.js

```diff
@@ -0,0 +1,74 @@
+var contentScoreName = 'content';
+var keywordScoreName = 'keyword';
+var notAvailableStatus = 'na';
+var scoreDescriptionClass = 'score-text';
+var imageScoreClass = 'image yoast-logo svg';
+
+(function( $ ) {
+	'use strict';
+	/**
+	 * Converts the first letter to uppercase in a string.
+	 * @returns {string} The string with the first letter uppercased.
+	 */
+	String.prototype.ucfirst = function () {
+		return this.charAt( 0 ).toUpperCase() + this.substr( 1 );
+	};
+
+	/**
+	 * Creates a text with the label and description for a seo score.
+	 * @param {String} scoreType The type of score, this is used for the label.
+	 * @param {String} status The status for the score, this is the descriptive status text.
+	 * @returns {String} A string with label and description with correct text decoration.
+	 */
+	function createSeoScoreLabel( scoreType, status ) {
```

Comment:
Generally we maintain the `var createSeoScoreLabel = function( scoreType, status )` notation when defining functions.

P.S. SEO should be capitalized ;)